tests/gen-test-certs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
This comes from the Redis source tree no, perhaps we can re-use that?

Reply:
I wanted to avoid depending on extra scripts. Assuming we don't need to clone redis ( we only do it in CI ) there is no need for any memtier developer to clone redis just to get the script. agree?

Reply:
After discussing with @yossigo this subject, we went a step further ron the simplification and we're now using redis-server from installed apt meaning no need to clone or build anything else apart from memtier on CI.